carp.protocols.core/src/commonTest/kotlin/dk/cachet/carp/protocols/domain/devices/BLEHeartRateTest.kt

```diff
@@ -0,0 +1,19 @@
+package dk.cachet.carp.protocols.domain.devices
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class BLEHeartRateTest {
+    @Test
+    fun registration_builder_sets_properties() {
+        val registration = BLEDeviceRegistrationBuilder().apply {
+            deviceId = "myid"
```

Comment:
I think deviceId was the common abstraction which is required to be unique. If the MAC acts as the unique deviceId simply set deviceId as the (verified/typesafe) MAC. Check AltBeaconDeviceRegistration for an example.

Reply:
I will redo the `BLEHeartRate` device descriptor and related stuff with this in mind.